guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>The <code>SuggestedBatchSize</code> property is also useful, as it specifies the optimal number of addresses to include in a single batch request. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>